discovery-infra/test_infra/logs_utils.py

```diff
@@ -67,7 +67,7 @@ def _verify_bootstrap_logs_uploaded(dir_path, file_path, installation_success):
     logs = gz.getnames()
     assert any("bootkube.logs" in s for s in logs), f"bootkube.logs isn't found in {logs}"
     if not installation_success:
-        for logs_type in ["dmesg.logs", "log-bundle"]:
+        for logs_type in ["dmesg.logs", "log-bundle", "journal.logs"]:
```

Comment:
`journal.logs` and `dmesg.logs` are collected on all nodes only if failure.

They both need to be added to `_verify_node_logs_uploaded` and add a `installation_success` to the function in order to test it only on failure.